R/get_regional_data.R

```diff
@@ -46,6 +46,9 @@ get_regional_data <- function(country, level = "1", totals = FALSE,
   region_class$download()
 
   # dataset specifc cleaning
+  if (verbose) {
```

Comment:
again thoughts on having this in clean (so it can contain a bit more info that is custom to that dataset?

Reply:
Sure, I will drop them from get national and regional and put a custom message in each country. Will also add to the template.